app/request_handler.py

```diff
@@ -29,6 +29,22 @@
 firebase = pyrebase.initialize_app(FIREBASE_CONFIG)
 db = firebase.database()
 
+def writeYelpRecords(yelpVenues):
+    record = {}
+
+    for biz in yelpVenues:
+        key   = representation.createKey(biz)
+        key   = key.encode('utf-8').strip()
```

Comment:
Same re: based off existing code, so I'm actually not sure why we're doing this. Any ideas? If not, I can remove it.